[2023-05-10T05:10:05.630+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: testing_stuff.run_remote_script scheduled__2023-05-10T05:00:00+00:00 [queued]>
[2023-05-10T05:10:05.640+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: testing_stuff.run_remote_script scheduled__2023-05-10T05:00:00+00:00 [queued]>
[2023-05-10T05:10:05.641+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-10T05:10:05.655+0000] {taskinstance.py:1350} INFO - Executing <Task(BashOperator): run_remote_script> on 2023-05-10 05:00:00+00:00
[2023-05-10T05:10:05.661+0000] {standard_task_runner.py:57} INFO - Started process 1062 to run task
[2023-05-10T05:10:05.667+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'testing_stuff', 'run_remote_script', 'scheduled__2023-05-10T05:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpomm1t8ah']
[2023-05-10T05:10:05.668+0000] {standard_task_runner.py:85} INFO - Job 55: Subtask run_remote_script
[2023-05-10T05:10:05.752+0000] {task_command.py:410} INFO - Running <TaskInstance: testing_stuff.run_remote_script scheduled__2023-05-10T05:00:00+00:00 [running]> on host 448394b5800f
[2023-05-10T05:10:05.926+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='testing_stuff' AIRFLOW_CTX_TASK_ID='run_remote_script' AIRFLOW_CTX_EXECUTION_DATE='2023-05-10T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-10T05:00:00+00:00'
[2023-05-10T05:10:05.930+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-05-10T05:10:05.933+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '#!/bin/bash\n\ncd ~\npwd']
[2023-05-10T05:10:05.958+0000] {subprocess.py:86} INFO - Output:
[2023-05-10T05:10:05.965+0000] {subprocess.py:93} INFO - /home/***
[2023-05-10T05:10:05.966+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-05-10T05:10:06.002+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=testing_stuff, task_id=run_remote_script, execution_date=20230510T050000, start_date=20230510T051005, end_date=20230510T051006
[2023-05-10T05:10:06.042+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-10T05:10:06.068+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
